src/styles/global.css

```
@@ -0,0 +1,1179 @@
+  margin: 0;
+  font-size: 0.74rem;
+}
+
+.pane-meta dt {
+  color: var(--gold);
+  text-transform: lowercase;
+}
+
+.pane-meta dd {
+  color: var(--foam);
+  word-break: break-word;
```

Stylelint (17.2.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)